ios/Mappers.swift

```diff
@@ -853,32 +853,50 @@ class Mappers {
         return list
     }
 
+    class func mapFormType(result: CollectInputsResult) -> String {
+        return if result is EmailResult {
+            "email"
+        } else if result is PhoneResult {
+            "phone"
+        } else if result is TextResult {
+            "text"
+        } else if result is NumericResult {
+            "numeric"
+        } else if result is SignatureResult {
+            "signature"
+        } else if result is SelectionResult {
+            "selection"
+        } else {
+            ""
+        }
+    }
+
     class func mapFromCollectInputsResults(_ results: [CollectInputsResult]) -> NSDictionary {
         var collectInputResults: [NSDictionary] = []
         for result in results {
             if result is EmailResult {
```

Comment:
we already have this check `if result is EmailResult` and don't need to call `mapFormType` as we are in the `if`block. We can directly set the formType to `email`. Same for other result types

Reply:
Instead of keeping the big `if else` growthing, given current each block is 80% the same, I'm wondering the possibility to extract that unique part in future pr and reduce duplication.